sdcm/cluster.py

```diff
@@ -2411,12 +2411,27 @@ def start_scylla(self, verify_up=True, verify_down=False, timeout=500):
         if verify_up:
             self.wait_jmx_up(timeout=timeout)
 
-    @retrying(n=3, sleep_time=5, allowed_exceptions=NETWORK_EXCEPTIONS + (CommandTimedOut, ),
+    @retrying(n=3, sleep_time=5, allowed_exceptions=NETWORK_EXCEPTIONS,
               message="Failed to stop scylla.server, retrying...")
     def stop_scylla_server(self, verify_up=False, verify_down=True, timeout=300, ignore_status=False):
         if verify_up:
             self.wait_db_up(timeout=timeout)
-        self.stop_service(service_name='scylla-server', timeout=timeout, ignore_status=ignore_status)
+        try:
+            self.stop_service(service_name='scylla-server', timeout=1, ignore_status=ignore_status)
```

Comment:
The one here is for testing I guess ?

Anyhow I'm not sure about the kill, it's o.k. that we'll have this info regardless if we'll have a coredump.

If something get stuck it will coredump, if just slow, we might not.